src/clib/xtg/surf_slice_cube.c

```diff
@@ -114,6 +114,8 @@ surf_slice_cube(int ncx,
 
     if (nmap != nslice) {
         logger_error(LI, FI, FU, "Something is plain wrong in %s (nmap vs nslice)", FU);
+        throw_exception("Something is plain wrong in surf_slice_cube (nmap vs nslice)");
```

Comment:
What would be the reason for the duplicate error message? When is it necessary to have both?